### PR TITLE
Add janzell to the list of contributors

### PR DESCRIPTION
**Pull request by janzell**

### CODE REVIEW

File: Data/member.json [hunk 1]
```diff
@@ -413,6 +413,13 @@
...
             "linkedinurl": "https://www.linkedin.com/in/syed-zuhair-hossain/"
-        }
+        },
+	    {
+	        "name": "Janzell Jurilla",
+	        "avatar": "https://avatars.githubusercontent.com/u/1677522?v=4",
+	        "description": "A passionate software developer from the Philippines", 
+	        "githuburl": "https://github.com/janzell",
+	        "linkedinurl": "https://www.linkedin.com/in/janzell/"
+    	}
```
Comment (NS007-dev) [inline Data/member.json]: Please can you move that forward a tab @janzell please?

File: Data/member.json [hunk 2]
```diff
@@ -413,6 +413,13 @@
             "description": "Bangladeshi | Studying Computer Science and Engineering at BRAC University, Dhaka | Webmaster, BRACU IEEE SB | Competitive Programmer |",
             "githuburl": "https://github.com/ZuhairHossain/",
             "linkedinurl": "https://www.linkedin.com/in/syed-zuhair-hossain/"
-        }
+        },
+	    {
```
Comment (NS007-dev) [inline Data/member.json:432]: @janzell after you move the bottom closing brace, please move the whole section you wrote one tab back, so its formatted like the rest of the file. Thank you.

**Review by NS007-dev — CHANGES_REQUESTED** [review]

**Review by NS007-dev — CHANGES_REQUESTED** [review]
@janzell you have conflicts, please can they be resolved, and if you need any help, feel free to ask. Thank you.

**Review by MrSlimCoder — APPROVED** [review]
I resolved conflicts by myself,
Thank you for contributing.

**Comment by MrSlimCoder** [comment]
@all-contributors add @janzell for code.

**Pull request merged**